assets/src/edit-story/elements/text/util.js

```diff
@@ -134,7 +134,8 @@ export const getHandleKeyCommand = (setEditorState) => (
 };
 
 export const generateFontFamily = (fontFamily, fontFallback) => {
-  let fontFamilyDisplay = fontFamily ? `${fontFamily}` : null;
+  // Wrap into " since some fonts won't work without it.
+  let fontFamilyDisplay = fontFamily ? `"${fontFamily}"` : null;
   if (fontFallback && fontFallback.length) {
     fontFamilyDisplay += fontFamily ? `,` : ``;
     fontFamilyDisplay += `${fontFallback.join(`,`)}`;
```

Comment:
What about the fallbacks here?

also, good opportunity to add some tests to document the expected behavior

Reply:
Fallbacks seem to work as expected, it looks like the fonts with numbers in the name specifically don't seem to work. Just with spaces seem to work fine (in my testing at least).

Reply:
Added a test for quotes.

Reply:
Won't fallback fonts ever have numbers? Doesn't it seem safer to just quote them all to be safe?

Reply:
Added for fallback, too: d7b7f7e

Reply:
Could you add testing the fallbacks, too?

(was a bit confused, but that's because the right commit of the fallback change is c42b30b, and not d7b7f7e as you wrote 😮)

Reply:
Oops, sorry for that! Will add a test directly for the util.